packages/wallet/test/index.spec.ts

```diff
@@ -27,617 +27,659 @@ const isRunningInKarma = (): boolean => {
+    'should return true if publicKey, privateKey pair is valid'
+
+  t.deepEqual(
+    fixtureWallet.verifyPublicKey(new Uint8Array(64)),
+    false,
+    'should return false if publicKey, privateKey pair is invalid'
+  )
+  t.end()
 })
 
-describe('public key only wallet', function () {
+tape('public key only wallet', (t) => {
   const pubKey = hexToBytes(fixturePublicKey)
-  it('.fromPublicKey() should work', function () {
-    assert.strictEqual(bytesToHex(Wallet.fromPublicKey(pubKey).getPublicKey()), fixturePublicKey)
-  })
-  it('.fromPublicKey() should not accept compressed keys in strict mode', function () {
-    assert.throws(function () {
+
+  t.deepEqual(
+    bytesToHex(Wallet.fromPublicKey(pubKey).getPublicKey()),
+    fixturePublicKey,
+    '.fromPublicKey() should work'
+  )
+
+  t.throws(
+    function () {
       Wallet.fromPublicKey(
         hexToBytes('030639797f6cc72aea0f3d309730844a9e67d9f1866e55845c5f7e0ab48402973d')
       )
-    }, /^Error: Invalid public key$/)
-  })
-  it('.fromPublicKey() should accept compressed keys in non-strict mode', function () {
-    const tmp = hexToBytes('030639797f6cc72aea0f3d309730844a9e67d9f1866e55845c5f7e0ab48402973d')
-    assert.strictEqual(
-      bytesToHex(Wallet.fromPublicKey(tmp, true).getPublicKey()),
-      '0639797f6cc72aea0f3d309730844a9e67d9f1866e55845c5f7e0ab48402973defa5cb69df462bcc6d73c31e1c663c225650e80ef14a507b203f2a12aea55bc1'
-    )
-  })
-  it('.getAddress() should work', function () {
-    assert.strictEqual(
-      bytesToHex(Wallet.fromPublicKey(pubKey).getAddress()),
-      'b14ab53e38da1c172f877dbc6d65e4a1b0474c3c'
-    )
-  })
-  it('.getPrivateKey() should fail', function () {
-    assert.throws(function () {
+    },
+    (err: any) => err.message.includes('Invalid public key'),
+    '.fromPublicKey() should not accept compressed keys in strict mode'
+  )
+
+  const tmp = hexToBytes('030639797f6cc72aea0f3d309730844a9e67d9f1866e55845c5f7e0ab48402973d')
+  t.deepEqual(
+    bytesToHex(Wallet.fromPublicKey(tmp, true).getPublicKey()),
+    '0639797f6cc72aea0f3d309730844a9e67d9f1866e55845c5f7e0ab48402973defa5cb69df462bcc6d73c31e1c663c225650e80ef14a507b203f2a12aea55bc1',
+    '.fromPublicKey() should accept compressed keys in non-strict mode'
+  )
+
+  t.deepEqual(
+    bytesToHex(Wallet.fromPublicKey(pubKey).getAddress()),
+    'b14ab53e38da1c172f877dbc6d65e4a1b0474c3c',
+    '.getAddress() should work'
+  )
+
+  t.throws(
+    function () {
       Wallet.fromPublicKey(pubKey).getPrivateKey()
-    }, /^Error: This is a public key only wallet$/)
-  })
-  // it('.toV3() should fail', function () {
+    },
+    (err: any) => err.message.includes('This is a public key only wallet'),
+    '.getPrivateKey() should fail'
+  )
+
+  // t.test('.toV3() should fail', function () {
   //   assert.throws(function () {
   //     Wallet.fromPublicKey(pubKey).toV3()
   //   }, /^Error: This is a public key only wallet$/)
   // })
```

Comment:
Why is this test commented out? can we just remove it?

Reply:
Not sure but I added it back in and it passes.